apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-library/src/main/java/gov/cms/model/dsl/codegen/library/DataTransformer.java

```diff
@@ -0,0 +1,775 @@
+package gov.cms.model.dsl.codegen.library;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+import com.google.protobuf.Timestamp;
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BooleanSupplier;
+import java.util.function.Consumer;
+import java.util.function.IntConsumer;
+import java.util.function.IntSupplier;
+import java.util.function.Supplier;
+import lombok.Data;
+import lombok.Getter;
+
+/**
+ * Stateful, mutable, non-thread safe object to facilitate transformation of data from incoming RDA
```

Comment:
Does this first sentence need to be revised, I don't think its limited to "RDA API objects" anymore.